### PR TITLE
🏷 Fix label remap

### PR DESCRIPTION
**Pull request by norkans7**
Fix https://sentry.io/nyaruka/textit-mailroom/issues/882996239/

https://sentry.io/nyaruka/textit/issues/876300089/

### CODE REVIEW

File: temba/flows/tests.py [hunk 1]
```diff
@@ -466,6 +466,19 @@ def test_flow_select2_response_with_exclude_flow_uuid(self):
...
 
+    def test_flow_import_labels(self):
+        self.assertFalse(Label.label_objects.all())
+
+        label = Label.get_or_create(self.org, self.admin, "Hello")
+        self.login(self.admin)
+        self.import_file("add_label")
+
+        flow = Flow.objects.filter(name="Add Label").first()
+        label_uuid_in_def = flow.revisions.first().definition["action_sets"][1]["actions"][0]["labels"][0]["uuid"]
```
Comment (norkans7) [inline temba/flows/tests.py]: the definition is the one used https://github.com/nyaruka/mailroom/blob/458586326941ae7ed334b42308cbe6a36d180664/models/flows.go#L164 and the first revision had an issue to remag labels but the label was created successfully from https://github.com/nyaruka/rapidpro/blob/432d541863978adc52ecb464a00641c5a8bcdb96/temba/flows/models.py#L2291

File: temba/flows/models.py [hunk 2]
```diff
@@ -1223,6 +1223,23 @@ def remap_uuid(json, attribute):
 
                 json[attribute] = new_uuid
 
+        def remap_label(label):
+            # labels can be single string expressions
+            if type(label) is dict:
```
Comment (dodobas) [inline temba/flows/models.py]: what do we do if the label is a single string expression? are we missing an `else` clause here

Comment (norkans7) [reply temba/flows/models.py]: Those are left as is and the expression will be evaluated at runtime

**Comment by norkans7** [comment]
The missing coverage is from https://github.com/nyaruka/rapidpro/blob/614b1e4b25f525c74b303829a417e7d0c4840316/temba/msgs/models.py#L1742

Not sure why we still have is_test in there? @rowanseymour any reason these were left in?

**Comment by rowanseymour** [comment]
@norkans7 I appear to have merged something into master that left that line uncovered.. not sure how that happened ... I'll remove that line

**Comment by nicpottier** [comment]
So to be clear upon a save this is fixed ya? Just wondering whether we need to do a migration to fix the flows that have already been migrated.

**Pull request merged**